src/main/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResource.java

```diff
@@ -79,6 +80,8 @@ public class CollectionResource {
 
   @Inject FeaturesConfig apiFeatureConfig;
 
+  @Inject OperationsConfig operationsConfig;
+
```

Comment:
+1 : can this and the   apiFeatureConfig be private ? Same on the other resources I think